sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -487,6 +487,25 @@ object SQLConf {
     .intConf
     .createWithDefault(10000)
 
+  val VECTORIZED_HUGE_VECTOR_RESERVE_RATIO =
+    buildConf("spark.sql.inMemoryColumnarStorage.hugeVectorReserveRatio")
+      .doc("spark will reserve requiredCapacity * this ratio memory next time. This is only " +
```

Comment:
```suggestion
      .doc("Spark reserves requiredCapacity * this ratio memory next time. This is only " +
```

Reply:
Can we avoid using the name of variable such as `requiredCapacity`? Also, it's a bit difficult to understand by reading the description here. I think you should explain this more explicitly including the behaviour of reserving * 2 by default.